standards/code-style.adoc

```diff
@@ -643,26 +650,30 @@ or `Fragment`.
 
 In the case of Activities the method is usually called `newStartIntent()`:
 
-[source, java]
+[source, kotlin]
 ----
-public static Intent newStartIntent(Context context, User user) {
-	Intent intent = new Intent(context, ThisActivity.class);
-	intent.putParcelableExtra(EXTRA_USER, user);
-	return intent;
+companion object {
+	fun newStartIntent(context: Context, user: User): Intent {
+	    val intent = Intent(context, ThisActivity::class.java)
+	    intent.putParcelableExtra(EXTRA_USER, user)
+	    return intent
+	}
 }
 ----
 
 For Fragments it is named `newInstance()` and handles the creation of the
 Fragment with the right arguments:
 
-[source, java]
+[source, kotlin]
 ----
-public static UserFragment newInstance(User user) {
-	UserFragment fragment = new UserFragment;
-	Bundle args = new Bundle();
-	args.putParcelable(ARGUMENT_USER, user);
-	fragment.setArguments(args)
-	return fragment;
+companion object {
+    fun newInstance(user: User) {
```

Comment:
This example could also make use of ```apply``` blocks similar to the Activity example.

Reply:
Updated.